x/ccv/provider/keeper/consumer_lifecycle.go

```diff
@@ -493,9 +512,16 @@ func (k Keeper) StopConsumerChain(ctx sdk.Context, consumerId string, closeChan
 	k.DeleteAllOptedIn(ctx, consumerId)
 	k.DeleteConsumerValSet(ctx, consumerId)
 
+	k.DeleteConsumerRewardsAllocation(ctx, consumerId)
+	k.DeleteConsumerOwnerAddress(ctx, consumerId)
+	k.DeleteConsumerRemovalTime(ctx, consumerId)
+
 	// TODO (PERMISSIONLESS) add newly-added state to be deleted
```

Comment:
Is this TODO still accurate?

Reply:
Yes. Precisely deciding what we should delete would need some time and it's not that important at this point. I think we can all agree that we need to keep consumer metadata, power-shaping, and initialization parameters.